src/nvim/channel.c

```diff
@@ -447,6 +449,7 @@ static uint64_t channel_create_internal_rpc(void)
 {
   Channel *channel = channel_alloc(kChannelStreamInternal);
   rpc_start(channel);
+  channel_create_event(channel, "RPC-internal");
   return channel->id;
```

Comment:
Also log this. Unless @bfredl intentionally did not want to log it for some reason?

Reply:
Good catch, omission due to early return in caller. I would rather refactor the caller to `goto end` and make it reuse the same `channel_create_event(channel, address)`.